package/MDAnalysis/analysis/align.py

```diff
@@ -1350,45 +1350,51 @@ def get_atoms_byres(g, match_mask=np.logical_not(mismatch_mask)):
 
             # stop if we created empty selections (by removing ALL residues...)
             if ag1.n_atoms == 0 or ag2.n_atoms == 0:
-                errmsg = ("Failed to automatically find matching atoms: created empty selections. " +
+                errmsg = ("Failed to automatically find matching atoms: created empty selections. "
                           "Try to improve your selections for mobile and reference.")
                 logger.error(errmsg)
                 raise SelectionError(errmsg)
-                
+
     if match_atoms:
         # check again because the residue matching heuristic is not very
         # good and can easily be misled (e.g., when one of the selections
         # had fewer atoms but the residues in mobile and reference have
         # each the same number)
-        try:
-            mass_mismatches = (np.absolute(ag1.masses - ag2.masses) > tol_mass)
-        except ValueError:
-            errmsg = ("Failed to find matching atoms: len(reference) = {}, len(mobile) = {} " +
-                    "Try to improve your selections for mobile and reference.").format(
-                        ag1.n_atoms, ag2.n_atoms)
-            logger.error(errmsg)
-            raise_from(SelectionError(errmsg), None)
-
-        if np.any(mass_mismatches):
-            # Test 2 failed.
-            # diagnostic output:
-            logger.error("Atoms: reference | trajectory")
-            for ar, at in zip(ag1[mass_mismatches], ag2[mass_mismatches]):
-                logger.error(
-                    "{0!s:>4} {1:3d} {2!s:>3} {3!s:>3} {4:6.3f}  |  {5!s:>4} {6:3d} {7!s:>3} {8!s:>3} {9:6.3f}".format(
-                        ar.segid,
-                        ar.resid,
-                        ar.resname,
-                        ar.name,
-                        ar.mass,
-                        at.segid,
-                        at.resid,
-                        at.resname,
-                        at.name,
-                        at.mass))
-            errmsg = ("Inconsistent selections, masses differ by more than {0}; "
-                    "mis-matching atoms are shown above.").format(tol_mass)
-            logger.error(errmsg)
-            raise SelectionError(errmsg)
+        if (not hasattr(ag1, 'masses') or not hasattr(ag2, 'masses')):
+            # # WARNING:
```

Comment:
Finally if you have remove this superfluous comment.

Reply:
I saw a comment of this kind somewhere else, therefore I wrote it.

Reply:
Having the comment there is not bad but I would say not needed because the code itself clearly says "warning".

However, I see a PEP8 issue below...